doc/source/whatsnew/v0.20.0.txt

```diff
@@ -41,6 +41,7 @@ Backwards incompatible API changes
 .. _whatsnew_0200.api:
 
 
+- ``CParserError`` has been aliased to ``ParserError`` in ``pd.read_csv`` and will be removed in the future (:issue:`12665`)
```

Comment:
Maybe say "renamed to" ? For me that sounds stronger that I have to change my code if I use it